animals/templates/fullanimallist.html

```diff
@@ -0,0 +1,54 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block pagetitle %}Current Animals{% endblock pagetitle %}
+
+{% block headertitle %}{% endblock headertitle %}
+
+{% block headersubtitle %}{% endblock headersubtitle %}
+
+{% block contenttitle %}
+<h2 class="section-title">Current animals <span>that need some help.</span></h2>
+<p>The list is below.</p>
+{% endblock contenttitle %}
+
+{% block bodycontent %}
+{% for a in animals %}
+    <!-- add a spacer element every 3rd object -->
+    {% if forloop.counter0 != 0 and forloop.counter0|divisibleby:3 %}<div class="spacer_50"></div>{% endif %}
```

Comment:
If you are using bootstrap you can just do 
<div class="row px-4"> 
{% for animal in animals %} 
<div class="col-md"> 
info 
</div> 
{% endfor %} 

This will add spacing to the y-axis, and since you didn't specify the columns widths it will auto size and correctly make sure that it fits good on the screen. 

https://getbootstrap.com/docs/4.0/utilities/spacing/

Again not that this solution is in correct, just offering my opinion for feedback.

Reply:
I'm no good at formatting haha. I just butchered the template so that there was a halfway working example. That spacer div was just what the example page used. I'm entirely open to cleaning that up using bootstrap.